content/docs/studio/user-guide/model-registry/assign-stage.md

```diff
@@ -67,7 +67,7 @@ assignment.
 
```

Comment:
The video included in this page (https://youtu.be/Vrp1O5lkWBo) must be updated, because the `Use --force …` checkbox is missing in the video. @aguschin are there any other changes needed in the video?

Reply:
models table and MDP have changed a lot, so it will be easier to re-record it completely I guess.